hazelcast/src/main/java/com/hazelcast/client/impl/operations/ClientDisconnectionOperation.java

```diff
@@ -30,7 +32,7 @@
 import java.util.Collection;
 import java.util.Set;
 
-public class ClientDisconnectionOperation extends Operation implements UrgentSystemOperation {
+public class ClientDisconnectionOperation extends Operation implements UrgentSystemOperation, IdentifiedDataSerializable {
```

Comment:
Isn't it more convenient to have a single super class 'ClientOperation' that takes care of the IdentifiedDataSerializable? This we can get rid of the duplication and create a bit more cohesion.